client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/index.tsx

```diff
@@ -9,9 +9,9 @@ import { useQuery } from 'calypso/landing/stepper/hooks/use-query';
 import { useSite } from 'calypso/landing/stepper/hooks/use-site';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { HostingBadge } from './hosting-badge';
+import { MigrationInstructions } from './migration-instructions';
```

Comment:
Renamed because it doesn't make sense to call it as `Sidebar` for the case without the preview where it will be the only content in the screen.